github/interactions_orgs.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
Would you mind bumping the Copyright year on new files (only) to 2019, please?
All existing files can keep their original Copyright year unchanged.
Thanks!